ImageFeed/Services/NetworkClient.swift

```diff
@@ -1,8 +1,13 @@
 import Foundation
 
 protocol NetworkRouting {
-    func fetch(url: URL, handler: @escaping (Result<Data, Error>) -> Void)
-    func fetch(request: URLRequest, handler: @escaping (Result<Data, Error>) -> Void)
+    @discardableResult func fetch(
+        url: URL, handler: @escaping (Result<Data, Error>) -> Void
+    ) -> URLSessionTask
```

Comment:
Если переносишь параметры, то переноси каждые два. Кажется это более менее последовательно.
```suggestion
    @discardableResult func fetch(
        url: URL, 
        handler: @escaping (Result<Data, Error>) -> Void
    ) -> URLSessionTask
```

Reply:
Поправил!